changelog/_unreleased/2025-07-29-fix-inconsistent-seourls-for-cross-selling-products.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Fix inconsistent seoUrls for cross-selling products
+issue: 11550
+---
+# Core
+* Added new method `Shopware\Core\Content\Seo\SalesChannel\SeoResolverData::getAll` to return all items with the given entity and identifier
+* Deprecated method `Shopware\Core\Content\Seo\SalesChannel\SeoResolverData::get`, please use `getAll` instead, as `get` returns only the first item which could lead to other items with the same entity and identifier being ignored
+___
+# Upgrade Information
+
+## Deprecate method Shopware\Core\Content\Seo\SalesChannel\SeoResolverData::get
```

Comment:
this should already mention the new method. so basically saying the `get` method is deprecated and will be removed with the next major and then this is the new alternative which should be used now. 

you can already add the before/after examples here. 

the `# Next Major Version Changes` then just states that the `get` method was removed and that they should use the `getAll` method instead. before/after could also be added here. 

be aware, that those sections will be extracted from this changelog file and go into two different upgrade guide files, first in the 6.7 file and second into the 6.8 file. 

so we kinda need to duplicate the content a bit. but it is still slightly different, as we introduce the deprecation first and then second remove it

Reply:
that's make sense 👍🏽 I updated it again